### DISCUSSION TITLE
Configurable default case mapper

### DISCUSSION THREAD

**Discussion by vpratfr** [spatie/laravel-data#750] [Ideas]
From what I understood from `NameMappersResolver.php`, the mapping strategy for attributes and classes comes from annotations.

Most of the time, in a project, naming conventions are (should be) consistent and hence we would use the same mapping for all data objects.

How would you configure that globally?

I may send a PR but I am not really confortable with performance implications. That should not be too hard to implement though.

**Comment by rubenvanassche** [comment]
You're welcome to send a PR, not a bad thing for in the config!
